hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestRootedOzoneFileSystemWithFSO.java

```diff
@@ -224,4 +225,19 @@ public void testDeleteVolumeAndBucket() throws IOException {
     Assert.assertTrue(deletes == prevDeletes + 1);
   }
 
+  @Test
+  public void testListStatusFSO() throws Exception {
+    Path parent = new Path(getBucketPath(), "testListStatusFSO");
```

Comment:
1300 can be extracted to a variable and provide details about why 1300 and why not 1023 ?